appkit/src/main/java/org/ergoplatform/appkit/RestApiErgoClient.java

```diff
@@ -14,26 +14,38 @@
     private final String _nodeUrl;
     private final NetworkType _networkType;
     private final ApiClient _client;
+    private final String _explorerUrl;
     private final ExplorerApiClient _explorer;
 
+    public static String defaultMainnetExplorerUrl = "https://api.ergoplatform.com";
```

Comment:
Thank you! I think these two constants could be final as well?

Reply:
thanks, fixed